src/lib/rule-finder.js

```diff
@@ -84,13 +84,22 @@ function _getAllAvailableRules(pluginRules) {
   return filteredAllRules
 }
 
-function RuleFinder(specifiedFile) {
+function _isNotCore(rule) {
+  return rule.indexOf('/') !== '-1'
+}
+
+function RuleFinder(specifiedFile, noCore) {
   var configFile = _getConfigFile(specifiedFile)
   var config = _getConfig(configFile)
   var currentRules = _getCurrentRules(config)
   var pluginRules = _getPluginRules(config)
-  var allRules = _getAllAvailableRules(pluginRules)
-  var unusedRules = difference(allRules, currentRules)
+  var allRules = pluginRules
+  if (noCore) {
+    currentRules = currentRules.filter(_isNotCore)
+  } else {
+    allRules = _getAllAvailableRules(pluginRules)
+  }
+  var unusedRules = difference(allRules, currentRules) // eslint-disable-line vars-on-top
 
```

Comment:
Yes, this prevents me from having to re-filter (conditionnally too) in each and every method below. Saves a few lines for each, and prevents copy-paste bugs.

In the case of the other methods, I get the point of doing the extra work in the method, in order to be lazy and prevent unneeded sorting, but in this case, the no-core filter will have to be done anyway. Unless the user is only requesting the plugin rules obviously, but IMO this is fine, as its cheap, and as said before, may prevent bugs and is more readable.